src/Weather.js

```diff
@@ -0,0 +1,11 @@
+class Weather {
+    currentWeather;
+    possibleWeather = ['sunny', 'sunny', 'stormy',]
+
+    checkWeather = () => {
+        let index = Math.floor(Math.random() * this.possibleWeather.length);
+        return this.currentWeather = this.possibleWeather[index]
```

Comment:
Which is better, the two lines above or:
return this.currentWeather = this.possibleWeather[Math.floor(Math.random() * this.possibleWeather.length)]

the lines written were easier for me to read and understand but would it be better to combine them into one line?